server/src/repositories/interview.repository.ts

```diff
@@ -166,30 +175,32 @@ export class InterviewRepository extends AbstractRepository<TaskChecker> {
   private async getInterviewPairs(courseTaskId: number): Promise<InterviewPair[]> {
     const result = await getManager().query(
       `
-SELECT
-  tc.id,
-  task_interview_result.score,
-  m_user."firstName" AS "interviewerFirstName",
-  m_user."lastName" AS "interviewerLastName",
-  m_user."githubId" AS "interviewerGithubId",
-  s_user."firstName" AS "studentFirstName",
-  s_user."lastName" AS "studentLastName",
-  s_user."githubId" AS "studentGithubId"
-FROM task_checker AS tc
-LEFT JOIN task_interview_result ON tc."studentId" = task_interview_result."studentId" AND tc."courseTaskId" = task_interview_result."courseTaskId"
-LEFT JOIN mentor AS m ON m.id = tc."mentorId"
-LEFT JOIN student AS s ON s.id = tc."studentId"
-LEFT JOIN "user" AS m_user ON m_user.id = m."userId"
-LEFT JOIN "user" AS s_user ON s_user.id = s."userId"
-WHERE tc."courseTaskId" = $1
+      SELECT
+        tc.id,
+        tir.score,
+        m_user."firstName" AS "interviewerFirstName",
+        m_user."lastName" AS "interviewerLastName",
+        m_user."githubId" AS "interviewerGithubId",
+        s_user."firstName" AS "studentFirstName",
+        s_user."lastName" AS "studentLastName",
+        s_user."githubId" AS "studentGithubId"
+      FROM task_checker AS tc
+      LEFT JOIN task_interview_result AS tir ON tc."studentId" = tir."studentId"
+                AND tc."courseTaskId" = tir."courseTaskId"
+                AND tc."mentorId" = tir."mentorId"
+      LEFT JOIN mentor AS m ON m.id = tc."mentorId"
+      LEFT JOIN student AS s ON s.id = tc."studentId"
+      LEFT JOIN "user" AS m_user ON m_user.id = m."userId"
+      LEFT JOIN "user" AS s_user ON s_user.id = s."userId"
+      WHERE tc."courseTaskId" = $1
     `,
       [courseTaskId],
     );
     return result.map((item: any) => {
       return {
         id: item.id,
         result: item.score,
-        status: item.score ? InterviewStatus.Completed : InterviewStatus.NotCompleted,
+        status: item.score || item.score === 0 ? InterviewStatus.Completed : InterviewStatus.NotCompleted,
```

Comment:
@aliakseitokarev I know that it will work like this, but for better readability please do
```suggestion
        status: (item.score || item.score === 0) ? InterviewStatus.Completed : InterviewStatus.NotCompleted,
```